plotly_resampler/aggregation/plotly_aggregator_parser.py

```diff
@@ -0,0 +1,158 @@
+import bisect
+from typing import List, Tuple, Union
+
+import numpy as np
+import pandas as pd
+
+from .aggregation_interface import DataAggregator, DataPointSelector
+
+
+class PlotlyAggregatorParser:
+    @staticmethod
+    def parse_hf_data(hf_data_dict, hf_keys: List[str]):
+        # TODO: check overhead --> add this to a hf_data parsing function
+        for k in hf_keys:
+            if k in hf_data_dict and isinstance(hf_data_dict[k], pd.Series):
+                # TODO: we can ommit pandas here, by using hasattr - see below
+                hf_data_dict[k] = hf_data_dict[k].values
+
+    @staticmethod
+    def to_same_tz(
+        ts: Union[pd.Timestamp, None], reference_tz
+    ) -> Union[pd.Timestamp, None]:
+        """Adjust `ts` its timezone to the `reference_tz`."""
```

Comment:
Can we define 'reference_tz' or include some information about what it means? If it's UTC, let's say that.

Reply:
Typed the `reference_tz` argument.